### PR TITLE
Forms Automated Test - LPS-131221 | master

### PR DESCRIPTION
**Pull request by ThalitaCeli**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 1]
```diff
@@ -32,13 +32,41 @@ definition {
...
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
+
+		Form.editFieldLabelMetal(fieldValue = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Test");
```
Comment (CalumR23) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:54]: You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?

**Comment by CalumR23** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 2]
```diff
@@ -32,13 +32,34 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be created"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeCreated {
 		property portal.acceptance = "false";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:37]: Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
@natocesarrego @diogocordeiro this is ready for review !

**Comment by brunofariasdeo** [comment]
Hi @ThalitaCeli,
I've just reviewed your pull request, great work! :)

**Comment by brunofariasdeo** [comment]
ci:forward

**Pull request closed without merging**